boards/x86/up_squared/doc/up_squared.rst

```diff
@@ -300,6 +299,135 @@ Steps
          WARNING: no console will be available to OS
 
 
+Booting the UP Squared Board over network
+=========================================
+
+Build Zephyr image
+------------------
+
+#. Follow `Build Zephyr application`_ steps to build Zephyr image.
+
+Prepare Linux host
+------------------
+
+#. Follow `Creating a GRUB2 Boot Loader Image from a Linux Host`_ steps
+   to create grub binary.
+
+#. Install DHCP, TFTP servers. For example `dnsmasq`
+
+   .. code-block:: console
+
+      $ sudo apt-get install dnsmasq
+
+#. Configure DHCP server. Configuration for `dnsmasq` is below:
+
+   .. code-block:: console
+
+      # Only listen to this interface
+      interface=eno2
+      dhcp-range=10.1.1.20,10.1.1.30,12h
+
+#. Configure TFTP server.
+
+   .. code-block:: console
+
+      # tftp
+      enable-tftp
+      tftp-root=/srv/tftp
+      dhcp-boot=grub_x86_64.efi
+
+   `grub_x86_64.efi` is a grub binary created above.
+
+#. Create the following directories inside TFTP root :file:`/srv/tftp`
+
+   :file:`EFI`
```

Comment:
Since you give the commands for the other steps, might as well to that here too (do these need a ``sudo``, and should we use ``$TFTP/`` instead of ``/srv/tftp/``?)

```
   .. code-block:: console

      $ mkdir -p /srv/tftp/EFI/BOOT
      $ mkdir -p /srv/tftp/kernel
``

Reply:
changed this way